extensions/blocks/opentable/opentable.php

```diff
@@ -9,55 +9,34 @@
 
 namespace Jetpack\OpenTable_Block;
 
-const FEATURE_NAME = 'opentable';
-const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
-
-/**
- * Check if the block should be available on the site.
- *
- * @return bool
- */
-function is_available() {
-	if (
-		defined( 'IS_WPCOM' )
-		&& IS_WPCOM
-		&& function_exists( 'has_any_blog_stickers' )
-	) {
-		if ( has_any_blog_stickers(
-			array( 'premium-plan', 'business-plan', 'ecommerce-plan' ),
-			get_current_blog_id()
-		) ) {
-			return true;
-		}
-		return false;
-	}
-
-	return true;
-}
+const FEATURE_NAME   = 'opentable';
+const BLOCK_NAME     = 'jetpack/' . FEATURE_NAME;
+const REQUIRED_PLANS = array( 'wpcom' => 'premium-plan' );
```

Comment:
IMO it would be valuable to generalize this slightly more. We might want to hide blocks for all users on either Jetpack or WPCOM. If we made both keys (`jetpack` and `wpcom`) required, then we could add either `true` or `false`, to simply enable/disable the block in these environments.

Reply:
Yeah, that's a good idea. We could leave it that you don't have to provide the key, but set it to `false` to turn them off because the blocks are available by default.